setup.py

```diff
@@ -0,0 +1,20 @@
+from setuptools import setup
+
+setup(
+    name='scrapy-crawlera',
+    version='1.0.1',
+    license='BSD',
+    description='Crawlera middleware for Scrapy',
+    author='Scrapinghub',
+    author_email='info@scrapinghub.com',
+    url='https://github.com/scrapinghub/scrapy-crawlera',
+    py_modules=['crawlera', 'hubproxy'],
+    platforms=['Any'],
+    classifiers=[
+        'Development Status :: 5 - Production/Stable',
+        'License :: OSI Approved :: BSD License',
+        'Operating System :: OS Independent',
+        'Programming Language :: Python',
+    ],
+    install_requires=['Scrapy>=0.22.0']
```

Comment:
I think we should remove Scrapy from install_requires because with it `pip install -U scrapy-crawlera` will auto-upgrade Scrapy, and this can break users projects.